### DISCUSSION TITLE
Remote controlled Ethernet cut off switch

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#18559] [General]
Hi,

I need a way to disconnect an Ethernet cable from remote now and then and thought about tasmota instantly.
While I could use a relay board, I'm a bit concerned that might interfere with Ethernet frequencies.
I also could use a PoE switch or other switch but that's overkill for a single connection here and there.

Does anyone know of a device that is tasmota capable that might do what I'm looking for?

**Comment by sfromis** [comment]
A simple solution, but not very elegant, would be a standard small Ethernet switch, with a relay to power it off, and on again after a suitable delay....

**Comment by dac2020** [comment]
Ha, that's a great idea and kind of elegant :). Thanks.

If anyone knows of anything purpose built, please let me know. In the meantime, I'm fishing out an old switch and using a tasmota plug.
